### PR TITLE
Deployment procedures updated

### PR DESCRIPTION
**Pull request by dongbohu**
This PR includes the following changes:
1. Renamed the long activity file `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt` into a more user-friendly filename `sample_signature_activity.txt`. Then changed all `Node###` strings into `Sig###` on the first row of this file. (The old one is moved to `old/` subdirectory as a backup. We can delete it when we are sure the new one is working.)
2. Renamed `node_gene_network.txt` into `signature_gene_network.txt`, also changed the `Node###` strings into `Sig###` in the first column of this file. (The old one is moved to `old/` subdirectory as a backup. We can delete it when we are sure the new one is working.)
3. Renamed `gene-gene_network_cutoff_0.2.txt.gz` to `gene_gene_network_cutoff_0.2.txt.gz`. 
4. Renamed `import_node_gene_network.py` into `import_signature_gene_network.py`.
5. Since `gene_gene_network_cutoff_0.2.txt.gz` is a superset of the original gene-gene network data file `eADAGE_net300_allNodes_ADAGEnet_PAID_corCutoff0.4.txt`, the latter has been moved to `old/` subdirectory too. We may delete it later.
6. Updated the default configuration file `adage/adage/config.py.template` because the filename changes above.
7. Updated the deployment script  `fabfile/adage_server.py`:  added a few steps to create a new machine learning model that has a lower gene-gene network cutoff value and import all mlmodel-related records into the database.
8. For some reason the permission of `import_data.py` was set to **755**, which is different from all the other management commands in that directory, so I set it to **644**. But the content is not changed at all, so you don't need to review it.

As you can see, (1) and (2) also fix issue #259.

@rzelayafavila: This PR may affect your docker deployment scripts. When this PR is closed, one of us will need to update docker deployment procedures too.

### CODE REVIEW

**Comment by dongbohu** [comment]
Here is a demo site whose signature names are all changed into **Sig###** format:
http://165.123.67.202/#/signature/search?mlmodel=7

**Comment by dongbohu** [comment]
I fixed all issues reported by codeclimate except that `Cyclomatic complexity is too high ...` one.

File: fabfile/adage_server.py [hunk 1]
```diff
@@ -190,13 +186,38 @@ def import_data_and_index():
...
+    mlmodel_basic = {
+        "title": "Ensemble ADAGE 300",
+        "cutoff": 0.4
+    }
+    mlmodel_complex = {
+        "title": "Ensemble ADAGE 300 with more complex gene-gene network",
+        "cutoff": 0.2
+    }
+    # Unzip gene-gene network file.
+    run('gunzip "%s.gz"' % CONFIG['data']['gene_network_file'])
```
Comment (rzelayafavila) [inline fabfile/adage_server.py]: I don't know how I feel about the `config.py.template` file stating that it is not a gzipped file and then this manually tacking on the ".gz" ending. Should there be a way to check here if this is a gzipped file?

File: fabfile/adage_server.py [hunk 2]
```diff
@@ -190,13 +186,38 @@ def import_data_and_index():
...
+    # Create two ML models and related records separately:
+    for mlmodel in [mlmodel_basic, mlmodel_complex]:
```
Comment (rzelayafavila) [inline fabfile/adage_server.py]: This is a cool way of doing this!

**Comment by rzelayafavila** [comment]
By the way @dongbohu, I saw your note, so I will integrate these changes into the documentation changes I am working on.

**Comment by dongbohu** [comment]
@rzelayafavila: I addressed the issue that you pointed out. Please take a look and let me know what you think. Thanks.

File: fabfile/adage_server.py [hunk 3]
```diff
@@ -190,13 +186,44 @@ def import_data_and_index():
...
+        "cutoff": 0.4
+    }
+    mlmodel_complex = {
+        "title": "Ensemble ADAGE 300 with more complex gene-gene network",
+        "cutoff": 0.2
+    }
+
+    # If the gene-gene network file has ".gz" extension, unzip it.
+    gene_gene_network_file = CONFIG['data']['gene_network_file']
+    basename, extension = os.path.splitext(gene_gene_network_file)
```
Comment (rzelayafavila) [inline fabfile/adage_server.py]: :+1:

File: fabfile/adage_server.py [hunk 4]
```diff
@@ -190,13 +186,44 @@ def import_data_and_index():
...
+    if extension == '.gz':
+        run('gunzip "%s"' % gene_gene_network_file)
+        gene_gene_network_file = basename
```
Comment (rzelayafavila) [inline fabfile/adage_server.py]: Nice!

**Pull request merged**